home/screens/ProjectsScreen.js

```diff
@@ -327,6 +328,12 @@ export default class ProjectsScreen extends React.Component {
         <Text style={styles.expoVersionText} onPress={this._copyClientVersionToClipboard}>
           Client version: {Constants.expoVersion}
         </Text>
+        <Text style={styles.supportSdksText}>
+          Supports SDK{Constants.supportedExpoSdks.length === 1 ? "" : "s"} {
+            // Displays the major version numbers of the SDKs in ascending order.
```

Comment:
Does this work as a comment? I think `//` comments are not supported inside JSX 🤔
Maybe try with `/* comment */`

Reply:
comments are `{/* ... */}` but I would just remove this comment altogether